src/dataflow/DataflowInferenceTreeAnnotator.java

```diff
@@ -132,7 +132,7 @@ private void replaceATM(AnnotatedTypeMirror atm, AnnotationMirror dataflowAM) {
         ab.setValue("value", cs.getId());
         AnnotationMirror varAnno = ab.build();
         atm.replaceAnnotation(varAnno);
-        atm.replaceAnnotation(dataflowAM);
+        // atm.replaceAnnotation(dataflowAM);
```

Comment:
In the end, please remove this line or add a comment highlighting why this isn't done.